README.md

```diff
@@ -29,7 +31,7 @@ To Visit App:
 
 ## Project Sceenshot(s)
 
- <p style="text-align:center;"><img src="https://github.com/EmeditWeb/aul-forum/blob/react-ts/client/public/forum.png" title="Aul web forum logo"></p>
+ <p style="text-align:center;"><img src="https://github.com/EmeditWeb/aul-forum/blob/react-ts/client/public/forum.png" title="Project screenshot"></p>
```

Comment:
Yeah this is good but I think it should be added towards the end of the project

Reply:
ooh yh,,you are right.